templates/zerver/integrations/git.md

```diff
@@ -8,23 +12,26 @@ Get Zulip notifications for your Git repositories!
 
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
+    Also update the value of `STREAM_NAME` to the name of your channel.
 
-    You can specify the branches that will be used for notifications by modifying
-    the `commit_notice_destination` function. By default,
-    pushes to the `main`, `master`, and `test-post-receive` branches will result in a
-    notification.
+    !!! tip ""
 
-1. Symlink `/usr/local/share/zulip/integrations/git/zulip_git_config.py`
-   to the `.git/hooks` directory of your Git repository.
+        Customize notification branches by editing the
+        `commit_notice_destination` function. By default,
+        pushes to the `main`, `master`, and `test-post-receive` branches
+        will result in a notification.
 
-1. Symlink `/usr/local/share/zulip/integrations/git/post-receive`
+1. Symlink both
```

Comment:
Merged these 2 steps.

Reply:
Can we give actual terminal commands to copy for this?